modules/openy_gc_auth/modules/openy_gc_auth_example/openy_gc_auth_example.libraries.yml

```diff
@@ -0,0 +1,5 @@
+openy_gc_auth_example:
+  js:
+    js/openy_gc_auth_example.js: {}
```

Comment:
let's ensure we have proper options - minified and load - see https://www.drupal.org/node/3281085

Reply:
Added. Thanks.